packages/mui-material/src/PigmentContainer/PigmentContainer.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { OverridableComponent, OverrideProps } from '@mui/types';
+// @ts-ignore
```

Comment:
To remove this `@ts-ignore` is far beyond this PR, it's related to the tsconfig setup with proper module resolution which I am not sure how to solve it.